frontend/common/src/components/UserProfile/ProfileSections/WorkPreferencesSection.tsx

```diff
@@ -64,7 +68,7 @@ const WorkPreferencesSection: React.FunctionComponent<{
   return (
     <Well>
       <div data-h2-flex-grid="base(flex-start, x2, x1)">
-        {wouldAcceptTemporary === null && (
+        {positionDuration === null && (
```

Comment:
Do we need to check for an empty array here?

Reply:
oh, yeah I think that would be important

https://github.com/GCTC-NTGC/gc-digital-talent/pull/4956/commits/72629730fa3fc394d4560478082d0eb0a85ceb4b